[ 2024-11-22 22:50:03,812 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 22:50:03,813 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 22:50:03,999 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 22:50:03,999 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 22:50:03,999 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 22:50:05,211 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 22:50:05,211 ] 68 utils.database - ERROR - Error executing query: not all arguments converted during string formatting
[ 2024-11-22 22:50:05,212 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 22:50:05,445 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 22:50:05,474 ] 26 __main__ - ERROR - Error fetching district data: Error executing query | Details: not all arguments converted during string formatting
[ 2024-11-22 22:52:47,614 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 22:52:47,615 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 22:52:47,618 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 22:52:47,618 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 22:52:47,619 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 22:52:48,041 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 22:52:48,041 ] 68 utils.database - ERROR - Error executing query: not all arguments converted during string formatting
[ 2024-11-22 22:52:48,041 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 22:52:48,061 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 22:54:40,162 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 22:54:40,163 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 22:54:40,167 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 22:54:40,168 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 22:54:40,168 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 22:54:40,733 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 22:54:40,733 ] 68 utils.database - ERROR - Error executing query: not all arguments converted during string formatting
[ 2024-11-22 22:54:40,733 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 22:54:40,753 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 22:57:32,710 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 22:57:32,711 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 22:57:32,715 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 22:57:32,715 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 22:57:32,715 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 22:57:33,198 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 22:57:33,198 ] 68 utils.database - ERROR - Error executing query: not all arguments converted during string formatting
[ 2024-11-22 22:57:33,198 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 22:57:33,216 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 23:00:20,749 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 23:00:20,750 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 23:00:20,753 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 23:00:20,753 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 23:00:20,754 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 23:00:21,380 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 23:00:21,380 ] 68 utils.database - ERROR - Error executing query: not all arguments converted during string formatting
[ 2024-11-22 23:00:21,381 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 23:00:21,402 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 23:00:21,436 ] 27 __main__ - ERROR - Error fetching data for district Bagalkot: Error executing query | Details: not all arguments converted during string formatting
[ 2024-11-22 23:02:43,267 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 23:02:43,268 ] 22 utils.database - INFO - Queries loaded successfully: ['import streamlit as st']
[ 2024-11-22 23:02:43,268 ] 74 utils.database - ERROR - Error executing query: Query 'FETCH_DISTRICT_DATA' not found in loaded queries.
[ 2024-11-22 23:02:43,269 ] 27 __main__ - ERROR - Error fetching data for district Bagalkot: Error executing query | Details: Query 'FETCH_DISTRICT_DATA' not found in loaded queries.
[ 2024-11-22 23:03:13,022 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 23:03:13,023 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 23:03:13,025 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 23:03:13,026 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 23:03:13,026 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 23:03:13,780 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 23:03:13,836 ] 74 utils.database - ERROR - Error executing query: 002049 (42601): 01b88bbd-0000-ce38-0008-4ae60002271a: SQL compilation error: error line 8 at position 17
Bind variable :district not set.
[ 2024-11-22 23:03:13,836 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 23:03:13,876 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 23:03:13,904 ] 29 __main__ - ERROR - Error fetching data for district Bagalkot: Error executing query | Details: 002049 (42601): 01b88bbd-0000-ce38-0008-4ae60002271a: SQL compilation error: error line 8 at position 17
Bind variable :district not set.
[ 2024-11-22 23:03:17,182 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 23:03:17,183 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 23:03:17,183 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 23:03:18,177 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 23:03:18,246 ] 74 utils.database - ERROR - Error executing query: 002049 (42601): 01b88bbd-0000-ce38-0008-4ae60002271e: SQL compilation error: error line 8 at position 17
Bind variable :district not set.
[ 2024-11-22 23:03:18,246 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 23:03:18,271 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 23:03:18,299 ] 29 __main__ - ERROR - Error fetching data for district Bagalkot: Error executing query | Details: 002049 (42601): 01b88bbd-0000-ce38-0008-4ae60002271e: SQL compilation error: error line 8 at position 17
Bind variable :district not set.
[ 2024-11-22 23:06:06,913 ] 21 logger - INFO - Logger is working correctly!
[ 2024-11-22 23:06:06,913 ] 22 utils.database - INFO - Queries loaded successfully: ['INSERT_USER_APPLICATION', 'FETCH_USERID_QUERY', 'UPDATE_USER_PROJECT_PROGRESS', 'FETCH_PENDING_APPLICATIONS', 'UPDATE_USER_STATUS', 'UPDATE_PMAY_DATA', 'UPDATE_USER_OFFICER_ID', 'FETCH_PROGRESS_NOT_STARTED', 'UPDATE_PMARY_PROGRESS_FOUNDATION', 'UPDATE_PMARY_PROGRESS_LINTEL', 'UPDATE_PMARY_PROGRESS_ROOF', 'UPDATE_PMARY_PROGRESS_COMPLETED', 'UPDATE_USER_FOUNDATION_URL', 'UPDATE_USER_LINTEL_URL', 'UPDATE_USER_ROOF_URL', 'UPDATE_USER_COMPLETED_URL', 'UPDATE_USER_SANITATION_URL', 'FETCH_USERS_WITH_COMPLETEDVERF', 'FETCH_TOTAL_BENEFICIARIES', 'FETCH_COMPLETED_HOMES', 'FETCH_UNSTARTED_PROJECTS', 'FETCH_STAGE_PROGRESS', 'FETCH_PERFORMANCE_RANKING', 'FETCH_DISTRICT_STAGE_PROGRESS', 'FETCH_DISTRICT_DATA', 'FETCH_DISTINCT_DISTRICTS']
[ 2024-11-22 23:06:06,916 ] 414 snowflake.connector.connection - INFO - Snowflake Connector for Python Version: 3.12.3, Python Version: 3.11.5, Platform: Windows-10-10.0.22631-SP0
[ 2024-11-22 23:06:06,916 ] 1197 snowflake.connector.connection - INFO - Connecting to GLOBAL Snowflake domain
[ 2024-11-22 23:06:06,916 ] 1278 snowflake.connector.connection - INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[ 2024-11-22 23:06:07,315 ] 42 utils.database - INFO - Connection to Snowflake established successfully.
[ 2024-11-22 23:06:07,368 ] 74 utils.database - ERROR - Error executing query: 002049 (42601): 01b88bc0-0000-cdc2-0008-4ae600021ab2: SQL compilation error: error line 8 at position 17
Bind variable :district not set.
[ 2024-11-22 23:06:07,368 ] 789 snowflake.connector.connection - INFO - closed
[ 2024-11-22 23:06:07,387 ] 795 snowflake.connector.connection - INFO - No async queries seem to be running, deleting session
[ 2024-11-22 23:06:07,416 ] 29 __main__ - ERROR - Error fetching data for district Bagalkot: Error occurred in script: C:\Users\riyaz\OneDrive\Desktop\PMAY\utils\database.py at line 61 - Error executing query
